server/@types/ui/index.d.ts

```diff
@@ -324,3 +324,5 @@ export type ApplicationSummaryData = {
     | 'CAS2 (formerly Bail Accommodation Support Services)'
     | null
 }
+
+export type AssessmentSearchApiStatus = 'unallocated' | 'in_review' | 'ready_to_place'
```

Comment:
Could you use this type and exclude the statuses that aren't needed? https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/main/src/main/resources/static/_shared.yml#L2400 
That way the statuses will always be in sync with the ones in the API

Reply:
Yup. I guess using `Extract` here may make more sense, as with `Exclude` any extra literals added to the API type would leak into this.